### DISCUSSION TITLE
Point-token

### DISCUSSION THREAD

**Discussion by PlyhSecret** [IBAX-io/go-ibax#14] [Typos （English）]
Hi mate, 
I will get 8500point. But, I don't know how to transfer point to token... 
thanks

**Comment by HalloKyle** [comment]
> Hi mate, I will get 8500point. But, I don't know how to transfer point to token... thanks

Redemption will start after the airdrop event. Patience my friend.

**Reply by PlyhSecret** [reply]
Thank mate, 
I hope will get more and more points to get many tokens...
